specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/AuthConfigs.json

```
@@ -174,6 +174,9 @@
         "x-ms-examples": {
           "Create or Update Container App AuthConfig": {
```

```suggestion
          "Create or Update Container App AuthConfig with sas url": {
```

---

this is the default auth config example without tokenStore

---

Could you add an explicit mention like 
```suggestion
          "Create or Update Container App AuthConfig without token store": {
```